BUILD

```diff
@@ -67,6 +67,8 @@ cc_library(
     hdrs = [
         ":gemmlowp_public_headers",
     ],
+    includes = ["."],
+    linkopts = ["-pthread"],
```

Comment:
It is a gcc-ism, but one that a lot of projects seem to suffer from. It's ignored with a warning on OS X.